docs/docs/hardware/input-devices/I2CRJ45.md

```diff
@@ -0,0 +1,44 @@
+---
+tags: ["I2CRJ45", "URC", "I2CRJ45"]
+---
+# I2CRJ45
+<p class="maker">by <b>SuperHouse Automation</b></p>
+
+<!-- Board Image -->
+![SuperHouse Automation I2CRJ45x8 board](/images/oxrs-I2CRJ45X8.jpg)
+![SuperHouse Automation I2CRJ45x16 board](/images/oxrs-I2CRJ45X16.jpg)
+![SuperHouse Automation I2CRJ45x24 board](/images/oxrs-I2CRJ45X24.jpg)
+
+<!-- Board Description -->
+I2C compactible baord that handles multiple inputs via an ethernet RJ45 connection
```

Comment:
compactible -> compatible
baord -> board
remove "ethernet" - it is just an RJ45 connection